### PR TITLE
2.8.0 release preparation

### PR DESCRIPTION
**Pull request by jwnasambu**

### CODE REVIEW

File: package/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	<groupId>org.openmrs.distro</groupId>
 	<artifactId>referenceapplication-package</artifactId>
-	<version>2.8.0-SNAPSHOT</version>
+	<version>2.8.0</version>
```
Comment (dkayiwa) [inline package/pom.xml]: Can you create a new pull request where you do not touch this file at all?

Comment (jwnasambu) [reply package/pom.xml]: Actually the current pull request i did exactly what you are suggesting

File: pom.xml [hunk 2]
```diff
@@ -32,50 +32,50 @@
...
 		<reportingcompatibilityVersion>2.0.5</reportingcompatibilityVersion>
 		<fhirVersion>1.13.0</fhirVersion>
-		<owaVersion>1.8.1</owaVersion>
-		<reportinguiVersion>1.5.0</reportinguiVersion>
+		<owaVersion>1.9.0</owaVersion>
+		<reportinguiVersion>1.6.0</reportinguiVersion>
 		<addresshierarchyVersion>2.11.0</addresshierarchyVersion>
 		<!-- OWAs -->
-		<sysadminVersion>1.0.1</sysadminVersion>
+		<sysadminVersion>1.1.0</sysadminVersion>
```
Comment (kaweesi) [inline pom.xml]: reset this version to 1.0.1 and run again, this should resolve the build

File: package/pom.xml (hunk 1 above)
Comment (kaweesi) [reply package/pom.xml]: @dkayiwa that may perhaps not build if other poms are having changed version or even the mismatch may be unbecoming in the tag, seems the conflict was with the change @jwnasambu had made on systemadmin owa version.
I have checked out her branch and rebuilt it until i zeroed to that area
 i see lots of module version changes as well, i hope these are ok?

File: package/pom.xml [hunk 3]
```diff
@@ -30,8 +30,8 @@
 
 	<build>
 		<plugins>
-			<plugin>
```
Comment (kaweesi) [inline package/pom.xml]: seems like this was a code mis-formatting

**Pull request closed without merging**